engine/server.py

```diff
@@ -461,30 +74,107 @@ async def initialize_project(base_dir: str) -> ReturnDict:
 
     @param base_dir: where is the project
     """
-    result = router_utils.init_project(base_dir=base_dir)
+    result = project_utils.init_project(base_dir=base_dir)
     return result
 
 
+@app.post("/get_base", tags=["project"])
+async def get_project_dir(task_id: str) -> ReturnDict:
+    """
+    get the project directory
+
+    """
+    return project_utils.get_project_dir(task_id=task_id)
+
+
+@app.post("/remove_task", tags=["project"])
+async def remove_task(task_id: str) -> ReturnDict:
+    """
+    remove task by task id
+
+    """
+    return project_utils.remove_task(task_id=task_id)
+
+
+@app.post("/list_projects", tags=["project"])
+async def list_project() -> ReturnList:
+    """
+    list all projects
+
+    """
+    return project_utils.list_projects()
+
+
+@app.post("/remove_project_by_id", tags=["project"])
+async def remove_project(task_id: str) -> ReturnDict:
+    """
+    remove the project
+
+    @param task_id: if for task
+    """
+    return project_utils.remove_project_dir(task_id=task_id)
+
+
+@app.post("/remove_project", tags=["project"])
+async def remove_project(project_name: str) -> ReturnStatus:
```

Comment:
this is the new remove project router, you can remove project via project name now

Reply:
what is the difference betwwen the two api

Reply:
user can delete the projects directly without init the project

Reply:
Alright, make sense!